core/domain/email_jobs_one_off_test.py

```diff
@@ -45,9 +45,8 @@ def _run_one_off_job(self):
         self.process_and_flush_pending_tasks()
 
     def test_hashes_get_generated(self):
-        # pylint: disable=unused-argument
         def _generate_hash_for_tests(
-                cls, recipient_id, email_subject, email_body):
+                cls, recipient_id, email_subject, email_body):  # pylint: disable=unused-argument
```

Comment:
Why do we have unused argument though?

Reply:
This is a test function and we are defining this method for swapping, so most of its arguments won't be needed.